.github/workflows/sonarscan.yml

```diff
@@ -49,15 +49,21 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '20'         
-          cache: 'yarn'
-          cache-dependency-path: './client/wfprev-war/src/main/angular/yarn.lock'
+          node-version: '20' 
+
+      - name: Add .npmrc file
+        working-directory: ./client/wfprev-war/src/main/angular
+        run: echo -e $NPMRC > ~/.npmrc
+        
+      - name: Debug .npmrc
+        run: cat ~/.npmrc 
      
       - name: Clear cache
         working-directory: ./client/wfprev-war/src/main/angular
         run: |
+          rm -f package-lock.json
           rm -rf node_modules
-          yarn cache clean
+          npm cache clean --force
```

Comment:
Why are we doing this? do you got yarn issue on sonar test build??

Reply:
Yes yarn was having issues finding lealfet dependencies for some reason. I changed to npm as I knew it was working for the other workflows, and didn't want to spend anymore time debugging.